src/H5FDsec2.c

```diff
@@ -1068,3 +1071,52 @@ H5FD__sec2_delete(const char *filename, hid_t H5_ATTR_UNUSED fapl_id)
 done:
     FUNC_LEAVE_NOAPI(ret_value)
 } /* end H5FD__sec2_delete() */
+
+/*-------------------------------------------------------------------------
+ * Function:    H5FD__sec2_ctl
+ *
+ * Purpose:     Sec2 VFD version of the ctl callback.
+ *
+ *              The desired operation is specified by the op_code
+ *              parameter.
+ *
+ *              The flags parameter controls management of op_codes that
+ *              are unknown to the callback
+ *
+ *              The input and output parameters allow op_code specific
+ *              input and output
+ *
+ *              At present, no op codes are supported by this VFD.
+ *
+ * Return:      Non-negative on success/Negative on failure
+ *
+ *-------------------------------------------------------------------------
+ */
+static herr_t
+H5FD__sec2_ctl(H5FD_t *_file, uint64_t op_code, uint64_t flags, const void H5_ATTR_UNUSED *input,
+               void H5_ATTR_UNUSED **output)
```

Comment:
Though this callback only handles the H5FD_CTL__TEST_OPCODE op code, I figured it would be good to have a ctl callback implemented in the default VFD for testing purposes.

Reply:
I feel like this should go in a testing VFD, not in the default VFD.

Reply:
Moved this into a minimal testing VFD in vfd.c